docs/BreakingChanges/063 Apps published with ClickOnce that use a SHA-256 certificate may fail on Windows 2003.md

```diff
@@ -0,0 +1,37 @@
+## 63: Apps published with ClickOnce that use a SHA-256 code-signing certificate.
```

Comment:
{
  "Id": "63",
  "Title": "Apps published with ClickOnce that use a SHA-256 code-signing certificate.",
  "Details": "The executable is signed with SHA256. Previously, it was signed with SHA1 regardless of whether the code-signing certificate was SHA-1 or SHA-256. This applies to:\n\n- All applications built with Visual Studio 2012 or later.\n- Applications built with Visual Studio 2010 or earlier on systems with the .NET Framework 4.5 present.\n\nIn addition, if the .NET Framework 4.5 or later is present, the ClickOnce manifest is also signed with SHA-256 for SHA-256 certificates regardless of the .NET Framework version against which it was compiled.",
  "Suggestion": "The change in signing the ClickOnce executable affects only Windows Server 2003 systems; they require that KB 938397 be installed.\n\nThe change in signing the manifest with SHA-256 even when an app targets the .NET Framework 4.0 or earlier versions introduces a runtime dependency on the .NET Framework 4.5 or a later version.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#ClickOnce",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis?"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "Only affects WinServer 2003; there is a KB to address it.  Could flag it if we detect click-once being used (especially if it has a SHA-1 code-signing cert)",
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis?